Odysee/Controllers/Content/FileViewController.swift

```diff
@@ -875,24 +919,37 @@ class FileViewController: UIViewController, UIGestureRecognizerDelegate, UINavig
     func loadTextContent(url: URL, contentType: String?) {
         DispatchQueue.global().async {
             do {
-                let contents = try String(contentsOf: url)
-                if contentType == "text/md" || contentType == "text/markdown" || contentType == "text/x-markdown" {
-                    guard let html = contents.markdownToHTML else {
-                        self
-                            .handleContentLoadError(String(
-                                format: String.localized("Could not load URL %@"),
-                                url.absoluteString
-                            ))
+                var request = URLRequest(url: url)
+                request.setValue("https://anthony.odysee.tv/", forHTTPHeaderField: "Referer")
```

Comment:
Referrer url here should also be updated.

Reply:
https://ios.odysee.com/

update bitwave > https://ios.odysee.com/